Added support for stock toolbar

linuxgurugamer opened the pull request:
For people who don't have the Blizzy toolbar installed, it now adds itself to the stock toolbar

linuxgurugamer committed:
Added support for stock toolbar

deram committed:
started developing osx support

deram committed:
osx build support for module

soulsource commented:
Would be awesome if that pull request would be merged in and the plugin would be officially rebuilt for KSP 1.4.1. 
The reason is, that on Linux KSP 1.4 does not work with analogue input (see [this Forum thread](https://forum.kerbalspaceprogram.com/index.php?/topic/171983-ksp-14-joystick-not-found-under-linux/) ) but Advanced Fly-By-Wire works just fine. 
It only needs to be recompiled against the new KSP and the new [Toolbar](https://forum.kerbalspaceprogram.com/index.php?/topic/161857-141-toolbar-continued-common-api-for-draggableresizable-buttons-toolbar/&).

deram committed:
Fixed indentation.

soulsource committed:
KSP 1.4 compatibility: We must push back the input we consume

soulsource committed:
Add condition to only push back events on Linux.

I think Unity on Windows does not use SDL, but I don't have a Windows box to confirm.

soulsource committed:
Change SDL event queue handling once more to keep the queue sorted.

linuxgurugamer committed:
Updated for 1.4.3

linuxgurugamer committed:
Adopted by LGG
	Updated for 1.4
	Added support for the ToolbarController
	Added support for the ClickthroughBlocker
	Removed option for Bizzy toobar, now handled by the Toolbarcontroller
	Added transparancy to large button

linuxgurugamer committed:
Updated jenkins

linuxgurugamer committed:
jenkins

linuxgurugamer committed:
Changed method of serializing settings
	Reorganized release directory

linuxgurugamer committed:
project

linuxgurugamer committed:
jenk

linuxgurugamer committed:
More jenkins

linuxgurugamer committed:
Added linux build

linuxgurugamer committed:
Rebuild for bad dll
	Moved XInputDotNetPure dll into Plugins folder
	Updated version

linuxgurugamer committed:
Added assembly dependency

linuxgurugamer committed:
Updated name in .version file
	Updated Jenkins file for Linux build
	Added missing file for Linux

linuxgurugamer committed:
Added needed dll

linuxgurugamer committed:
Merge branch 'osx_build' of https://github.com/deram/ksp-advanced-flybywire into deram-osx_build

linuxgurugamer committed:
Merged in OSX branch

linuxgurugamer committed:
Merge branch 'feature/KSP1.4' of https://github.com/soulsource/ksp-advanced-flybywire into soulsource-feature/KSP1.4

linuxgurugamer committed:
Merge branch 'soulsource-feature/KSP1.4' into dev

linuxgurugamer committed:
Merge pull request #3 from linuxgurugamer/dev

Dev

linuxgurugamer committed:
Updated the SDLs library
	Updated the SDL2 file
	Added code to dynamically get the FSM offsets:  InitFsmOffsets
	Updated events to use dynamic FSM offsets

linuxgurugamer committed:
Updated SDL2.dll

linuxgurugamer committed:
Merge branch 'dev'

linuxgurugamer committed:
Merge branch 'master' of https://github.com/linuxgurugamer/ksp-advanced-flybywire

linuxgurugamer committed:
Updated the SDLs library
	Updated the SDL2 file
	Added code to dynamically get the vectorField and floatField offsets
	Rewrote all event code for EVA events to use new methods of calling
	Repositioned the GUI.DragWindow() calls to allow full windows to be draggable
	Added a DragWindow to the Choose Action windows, and fixed the call to properly save the location

linuxgurugamer committed:
Changelog update

linuxgurugamer committed:
Added jenkins for osx
Updated jenkins-win

linuxgurugamer committed:
More jenkins updates

linuxgurugamer committed:
Updated csproj

linuxgurugamer committed:
Added back the SDL2.dll

linuxgurugamer committed:
Updated projkect files

linuxgurugamer committed:
updated SDL2.cs for Linux and OSX

linuxgurugamer committed:
Updated version

linuxgurugamer committed:
Fixed builds
	Added option to run at max speed always when autorun is enabled
	Now closes the config window after saving
	Tweaked size of some buttons
	Some minor optimizations
	Now allows deleting 1st preset in list, if more than one exists

linuxgurugamer committed:
Centered Mod Settings button
	Fixed RadialIn/RadialOut buttons (are reversed)
	Fixed Pitch Trim +/- to adjust trim by .1 per click

linuxgurugamer committed:
Added option to allow throttle to be overridden by joystick or not
 Fixed loading of settings upon entering the flight scene
 Fixed saving of:
  m_MaxMoveSpeedAlways
  m_UseOnPreInsteadOfOnFlyByWire

linuxgurugamer committed:
Version bump for 1.5 rebuild

linuxgurugamer committed:
Refreence update

linuxgurugamer committed:
Removed KSP_VERION_MAX and KSP_VERSION

linuxgurugamer committed:
Updated BUILD version

linuxgurugamer committed:
Updated BUILD version

linuxgurugamer committed:
Added .github/FUNDING.yml

linuxgurugamer committed:
Fixed controller affecting multiple vessels at a time
 Moved textures out of PluginData directory
 Updated AssemblyVersion.tt

linuxgurugamer committed:
Updated changelog

linuxgurugamer committed:
Added code to create data folder (PluginData) if it doesn't exist

linuxgurugamer committed:
Updated version file

linuxgurugamer committed:
Added InstallChecker
 Updated for KSP 1.8

bhaan committed:
fixed null reference exception for empty presets

bhaan committed:
fixed xml serialzation

bhaan committed:
updated example config

bhaan committed:
re-enable nav-ball toggle (speed mode)

bhaan committed:
remove set sas ui function

bhaan committed:
add sas invert to descrete action list

linuxgurugamer committed:
Merge pull request #20 from bhaan/null-ref-bug

Fix Xml Serialization for KSPv1.8

linuxgurugamer committed:
Merge pull request #21 from bhaan/speed-mode

re-enable nav-ball toggle (speed mode)

linuxgurugamer committed:
Merge pull request #22 from bhaan/sas-invert

New Feature: SAS Invert

linuxgurugamer committed:
Fixed initialization of variable outside of MonoBehaviour loop

linuxgurugamer committed:
Updated README with correct URL for download, and short OSX install section

Commit:
Update installation instructions and links

Fix broken link to forum page. Remove broken KerbalStuff link. Add Spacedock
links.

Move dependencies into installation instructions. Move OSX instructions from
bottom of readme.

Update links that go to AlexanderDzhoganov's Github so they point to this repository.

linuxgurugamer committed:
Merge pull request #27 from jelaby/master

Update installation instructions and links

linuxgurugamer committed:
Add right-click to disable/enable any active controllers
 Inverted the button colors when disabled
 Reduced size of main window

linuxgurugamer committed:
Merge branch 'master' of https://github.com/linuxgurugamer/ksp-advanced-flybywire

linuxgurugamer committed:
Renamed DLL for CKAN compatibility
 Added AssemblyFileVersion
 Updated version file for 1.12

linuxgurugamer committed:
Restored missing DLLs

linuxgurugamer committed:
Updated SDL2 for KSP 1.12
 For now, no support for Linux or Mac

linuxgurugamer committed:
Updated for linux & Mac

linuxgurugamer committed:
removed bad DLLs

Dessix committed:
Add support for custom axis groups

linuxgurugamer committed:
Fixed small memory leak

linuxgurugamer committed:
Merge pull request #39 from Dessix/dessix/groundbreaking

Add support for Custom Axis Groups

linuxgurugamer committed:
Updated changelog and version

linuxgurugamer committed:
Fixed issue where buttons to change throttle weren't working unless there was at least one axis defined

Commit:
add check for vessel control state

Commit:
added checks for other buttons

khaddra committed:
Change libSDL2 to generic version on linux

linuxgurugamer committed:
Merge pull request #46 from archiecarrot123/master

Change libSDL2 to generic version on linux

linuxgurugamer committed:
Thanks to github user @archiecarrot123 for this:
  On Linux, changes the libSDL2 version to a generic version (fixes compatibility with different versions of libSDL2)
 Updated KSP versions in the version file, and URL for mod reference

linuxgurugamer committed:
slight edit to changelog

linuxgurugamer committed:
minor edit

linuxgurugamer committed:
minor chagne

linuxgurugamer committed:
minor edit

khaddra committed:
Change libSDL2 to libSDL2-2.0.so.0

linuxgurugamer committed:
Merge pull request #47 from archiecarrot123/master

Change libSDL2 to libSDL2-2.0.so.0

linuxgurugamer committed:
Merge branch 'master' of github.com:linuxgurugamer/ksp-advanced-flybywire

linuxgurugamer committed:
Updated Changelog and version

linuxgurugamer committed:
Fixed version number in changelog

linuxgurugamer committed:
Merge branch 'Jonathon594-2' into master

linuxgurugamer committed:
Merge branch 'Jonathon594-master' into Jonathon594-3

linuxgurugamer committed:
Updated Changelog & version file

linuxgurugamer committed:
Merge pull request #55 from linuxgurugamer/Jonathon594-3

		Add check for vessel control state for the flight control axises.  does not affect action groups

NateSaxo committed:
Change libSDL2-2.0.14 to libSDL2-2.0.0 for osx

linuxgurugamer committed:
Merge pull request #58 from NateSaxo/master

Change libSDL2-2.0.14 to libSDL2-2.0.0 for osx

linuxgurugamer committed:
Thanks to github user @NateSaxo for this:

		Change libSDL2-2.0.14 to libSDL2-2.0.0 for OS X
	Localized the strings in the mod
